server/src/main/scala/com/all4journey/webapp/pages/ProfilePage.scala

```diff
@@ -12,6 +12,7 @@ import prickle.Unpickle
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.{Future, Await}
 import scala.concurrent.duration.DurationInt
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/rjkernick/all4journey/file/2445603283/issues/source?bid=3246460&fileBranchId=3246461#l14)